dockerfiles/dataset/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:14.04
+MAINTAINER Madhuri Yechuri <madhuri.yechuri@clusterhq.com>
+
+# Note: apt-get install requires the `-1` (epoch) suffix.
+ENV FLOCKER_VERSION 1.14.0-1
+
+RUN sudo apt-get update
+RUN sudo apt-get -y install apt-transport-https software-properties-common
+RUN sudo add-apt-repository -y ppa:james-page/docker
+RUN sudo add-apt-repository -y "deb https://clusterhq-archive.s3.amazonaws.com/ubuntu/$(lsb_release --release --short)/\$(ARCH) /"
```

Comment:
:question: If the repo was configurable -- by way of an ENV variable -- we'd be most of the way towards being able to build Docker images for any version of Flocker. (in a followup branch)

See 
  * https://github.com/ClusterHQ/flocker/blob/master/build.yaml#L488 (and surrounding code

...for an idea of how we configure Jenkins to build packages, create a repo and then point `run-acceptance-tests --build-server` at a repository server running on the build slave:
 * https://github.com/ClusterHQ/flocker/blob/master/build.yaml#L433